src/api/java/com/minecolonies/api/compatibility/CompatibilityManager.java

```diff
@@ -355,7 +355,10 @@ public ItemStack getSaplingForLeaf(final BlockState block)
     @Override
     public List<ItemStorage> getCopyOfPlantables()
     {
-        return ImmutableList.copyOf(plantables);
+        return ImmutableList.copyOf(plantables)
+                 .stream()
```

Comment:
I mean, we should prevent them getting into this list in the first place.

Reply:
No need to filter every call the list.

Reply:
Ok, I see now. `discoverablePlantables` refers to compostable items but its beecause it w was copied from compostable not that it used them. I will fix.